tests/client/core/api/test_api.js

```diff
@@ -1,4 +1,5 @@
 import * as api from 'core/api';
+import * as helpers from 'tests/client/helpers';
```

Comment:
I'll change it to the selective import since it's pretty easy to glance at the imports.